app/views/passengers/index.haml

```diff
@@ -64,7 +65,7 @@
           %th
     %tbody
       - @passengers.each do |passenger|
-        %tr{class: expiration_check(passenger)}
+        %tr{class: passenger.doctors_note.present? ? expiration_check(passenger.doctors_note) : ""}
```

Comment:
I think it would be best to just call the method and check for nil in the method itself so we can handle it better.